pygus/gus/models.py

```diff
@@ -183,7 +217,7 @@ def step(self):
         # print(self.release_bins['slow'])
         # print(self.release_bins['fast'])
 
-    def _load_experiment_parameters(self, experiment):
+    def _load_experiment_parameters(self, experiment: Dict):
```

Comment:
@oguzhan @bulentozel 
Is there a reason why the "scenario.json" is expected as a Dict, while the `species_composition` and `allometrics` files are expected as filepaths and parsed from JSON in the library?